fixtures/kelvin/epos-300.json

```diff
@@ -0,0 +1,894 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Epos 300",
+  "categories": ["Dimmer", "Color Changer", "Strobe"],
+  "meta": {
+    "authors": ["Bjorn"],
+    "createDate": "2023-10-09",
+    "lastModifyDate": "2023-10-09"
+  },
+  "links": {
+    "manual": [
+      "https://www.kelvinlight.com/app/uploads/2023/08/Epos-300-User-Manual-EN-V1.0-May-2023-2.pdf",
+      "https://github.com/KelvinLights/Kelvin-DMX-Protocol/"
+    ],
+    "productPage": [
+      "https://www.kelvinlight.com/product/epos_300_full_color_spectrum_rgbacl_led_studio_light/"
+    ]
+  },
+  "physical": {
+    "dimensions": [17, 33, 13],
+    "weight": 7,
+    "power": 300,
+    "DMXconnector": "5-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "Main intensity": {
+      "name": "Main intensity",
+      "defaultValue": 0,
+      "fineChannelAliases": ["Main intensity fine"],
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Output type 8bit": {
+      "name": "Output type",
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "type": "Generic",
+          "dmxRange": [0, 0],
+          "comment": "Constant output"
+        },
+        {
+          "type": "Generic",
+          "dmxRange": [1, 255],
+          "comment": "Maximum output"
+        }
+      ]
+    },
+    "CCT": {
+      "name": "CCT",
+      "defaultValue": 0,
+      "fineChannelAliases": ["CCT fine"],
+      "capability": {
+        "type": "ColorTemperature",
+        "colorTemperatureStart": "1700K",
+        "colorTemperatureEnd": "20000K"
+      }
+    },
+    "Green-magenta point 8bit": {
+      "name": "Green-magenta point",
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "type": "Generic",
+          "dmxRange": [0, 124],
+          "comment": "Neutral"
+        },
+        {
+          "type": "Generic",
+          "dmxRange": [125, 255],
+          "comment": "-0.027 to 0.027"
+        }
+      ]
+    },
+    "Green-magenta point 16bit": {
+      "name": "Green-magenta point",
+      "fineChannelAliases": ["Green-magenta point 16bit fine"],
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "type": "Generic",
+          "dmxRange": [0, 31906],
+          "comment": "Neutral"
+        },
+        {
+          "type": "Generic",
+          "dmxRange": [31907, 65535],
+          "comment": "-0.027 to 0.027"
+        }
+      ]
+    },
```

Comment:
You don't need to specify both an 8bit and a 16bit version of a channel. Simply specify the 16bit version (and just call it "Green-magenta point" and "Green-magenta point fine") and only use the main "Green-magenta point" channel without the "Green-magenta point fine" channel in an 8bit mode. (The same applies also to other channels.)

Reply:
Thanks for the help. I am trying to find example fixtures to help me implement this.  I did it similarily to how its done for the ARRI fixtures, but those fixture files are outdated then?  Is it ok that there is no dmxRange capabilities specified for 8 bit ?

Reply:
16-bit DMX ranges are automatically down-scaled when only the 8-bit channel is used. Maybe for the ARRI fixtures, the channels are separated because the down-scaled capability ranges don't match the actual ones?

I currently don't have an example fixture in my head; see also this issue I just discovered:

* https://github.com/OpenLightingProject/open-fixture-library/issues/3608